packages/babel-core/src/transformation/util/clone-deep.ts

```diff
@@ -1,8 +1,13 @@
 import v8 from "v8";
-import cloneDeep from "./clone-deep-browser";
+import cloneDeep from "lodash.clonedeep";
+import semver from "semver";
 
 export default function (value) {
-  if (v8.deserialize && v8.serialize) {
+  if (
+    v8.deserialize &&
+    v8.serialize &&
+    semver.satisfies(process.versions.node, ">=18.2.0")
```

Comment:
It was fixed in node 18.2.

Reply:
Is there a Node.js or V8 bug report about this somewhere?

Reply:
https://github.com/nodejs/node/pull/42695